esi_leap/common/statuses.py

```diff
@@ -20,4 +20,7 @@
 WAIT_EXPIRE = 'wait expire'
 WAIT_FULFILL = 'wait fulfill'
 
+OFFER_CAN_DELETE = ('available', 'error')
+LEASE_CAN_DELETE = ['active', 'created', 'error']
```

Comment:
LEASE_CAN_DELETE and CANCELLABLE_LEASE_STATUSES can probably be combined (I like your variable name better!). Just make sure it contains all statuses from both.

Reply:
Okay, will do!